### PR TITLE
Allow scrolling unless long press on Android.

### PR DESCRIPTION
**Pull request by joshuapinter**
Fixes one of the big ToDos in https://github.com/gitim/react-native-sortable-list/issues/15.

### CODE REVIEW

File: src/Row.js [hunk 1]
```diff
@@ -40,8 +40,21 @@ export default class Row extends Component {
...
+      if (this._isDisabled()) return false;
+
+      return true;
+    },
+
+    onMoveShouldSetPanResponder: () => {
+      if (this._isDisabled()) return false;
+
+      return true;
+    },
```
Comment (gitim) [inline src/Row.js]: this change is not required, right?

Comment (joshuapinter) [inline src/Row.js]: Not required, I just think it reads better than `return !this._isDisabled()`. 

Less mental gymnastics you have to do.

**Comment by gitim** [comment]
cool! thanks!

**Comment by joshuapinter** [comment]
👍

**Pull request merged**